owncloud/indico_owncloud/client/index.js

```diff
@@ -17,6 +19,12 @@ window.setupOwncloudFilePickerWidget = ({filepickerUrl, fieldId}) => {
       message.data.files
     ) {
       document.getElementById(`${fieldId}-files`).value = message.data.files.join('\n');
+
+      if (!message.data.ready || !message.data.files.length) {
+        submitButton.setAttribute('disabled', '');
```

Comment:
`submitButton.disabled = <bool>;`

Reply:
I've seen that if I set it to a bool, it will add `disabled="true"` to the DOM, but isn't just `disabled` more correct?

Reply:
https://stackoverflow.com/a/23701672/298479

Setting the property is more convenient (bool) and controls the attribute internally.